mapeditor/mapview.cpp

```diff
@@ -270,6 +270,7 @@ void MapView::mousePressEvent(QMouseEvent *event)
 			if(event->button() == Qt::LeftButton)
 			{
 				auto * obj = sc->selectionObjectsView.selectObjectAt(tileStart.x, tileStart.y);
+				auto * obj2 = sc->selectionObjectsView.selectObjectAt(tileStart.x, tileStart.y, obj);
```

Comment:
I may be missing context, but what is obj & obj2? 
What if we have more than 2 objects on a tile?
I think this place deserves a comment to explain why such logic is needed

Reply:
exactly my thoughts